examples/gocean/eg5/README.md

```diff
@@ -32,25 +33,26 @@ libraries yourself, and modify the ``Makefile`` in this directory
 to specify the required linking parameters. The ``Makefile``
 supports the following environment variables that can be defined
 to find the various software packages:
-DL_DIR:
+
+### DL_DIR:
    The location of the dl_esm_inf library, it defaults to
    ``../../../external/dl_esm_inf/finite_difference/src``,
    which is the version included in PSyclone.
-DL_TIMER_DIR:
+### DL_TIMER_DIR:
     The location of the apeg-dl_timer library. It defaults to
-    ``../../../../apeg-dl_timer``, i.e. it assumes that apeg-dl_timer
+    ``../../../../apeg-dl_timer``, i.e. it is assumed that apeg-dl_timer
     is installed next to PSyclone.
-DRHOOK_DIR:
+### DRHOOK_DIR:
     The location of DrHook. It defaults to
-    ``../../../../drhook``, i.e. it assumed that DrHook is
+    ``../../../../drhook``, i.e. it is assumed that DrHook is
     installed next to PSyclone.
 
 The makefile here will invoke psyclone with the ``--profile invokes``
 flag, which will add profiling around the two invokes used in the example.
 
-.. Note:
-    The actual runtime is extremly short, so likely the profiling
-    tool used will report 0 seconds for each of the invokes.
+### Note:
```

Comment:
I think `.. note:` would be better as Note isn't really a section.

Reply:
That is sphinx makeup, github does not understand it and renders it as plain text - that's why I changed it. I couldn't find any markup more suitable than section (suggestions welcome).

Reply:
Oops! Ignore me :-)